lensterxyz/lenster
# feat: add message preview component

## What does this PR do?
This PR maps Lens profiles to XMTP accounts and only shows messages for conversations that include a valid Lens profile.

This is a work in progress and behind the `messages` feature flag. Some known issues include:
- Right now the preview shows the first message not the most recent
- The conversations in the left panel are not sorted correctly
- Pieces are duplicated between Message and Messages. They should be pulled out into their own `ConversationList` component.

<img width="1414" alt="Screen Shot 2022-10-10 at 2 57 29 PM" src="https://user-images.githubusercontent.com/556051/194935002-ec104814-dbed-47c3-ba61-0a04dfd618c4.png">


## How should this be tested?

- Ensure you're in the FF and navigate to `Messages`
- You should see a list of all your current DM conversations with Lens profiles
- Tapping the cell should open the message list component
